⚗️ Modify variable and property names

AlejandroPad opened the pull request:
- [x] Cambiamos a inglés los nombres de las propiedades de `tests/data/logistic_model_parameters_tests.json`. 
- [x] Detectamos que al cambiarlas, surge un error en las pruebas con los valores esperados (al intentar hacer lo análogo en `data/processed/logistic_model_parameters.json` al correr las pruebas se vuelven a cambiar automaticamente a su forma Spanglish). 
- [x] Probar con otro json file generado con los datos de prueba
## Hecho
- Extrajimos la función `get_best_json_for_logistic_model` 
- Factorizamos funciones pequeñas en `get_best_json_for_logistic_model`
## Pendientes
- [ ] Eliminar json de la receta de los tests en `Makefile`
- [ ] Extraer funciones de los `src`

Commit:
⚗️ Modify variable and property names

Se cambió a inglés los nombres de las propiedades de `logistic_model_parameters_tests.json`. Igualmente se detectó que al cambiarlas, surge un error en las pruebas con los valores esperados.

Commit:
⚗️Modify property names

Se cambiaron los nombres de las propiedades a inglés.

Commit:
:alembic: Modify property order and values for testing

Se cambió el orden de donde debe de ir `skull_length` al igual que su valor en la prueba.

Commit:
🚧Add `obtained_json` function to R file

Se agregó la función `obtained_json` con el fin de no utilizar toda la población en las pruebas.

devarops commented on data/raw/datapackage.json:
```diff
@@ -169,7 +169,7 @@
                     },
                     {
-                        "name": "longitud_ala_cerrada",
+                        "name": "close_brim_length",
```
```suggestion
                        "name": "closed_wing_length",

```

devarops replied:
```suggestion
                        "name": "closed_wing_length",

```

devarops commented on data/raw/datapackage.json:
```diff
@@ -181,7 +181,7 @@
                     },
                     {
-                        "name": "longitud_ala_abierta",
+                        "name": "open_brim_length",
```
```suggestion
                        "name": "opened_wing_length",

```

devarops commented on data/raw/datapackage.json:
```diff
@@ -193,9 +193,9 @@
                     },
                     {
-                        "name": "media_envergadura",
+                        "name": "media_wingspan",
```
```suggestion
                        "name": "half_wingspan",

```

devarops commented on data/raw/datapackage.json:
```diff
@@ -193,9 +193,9 @@
+                        "name": "media_wingspan",
                         "long_name": "Half wingspan",
-                        "nombre_largo": "Media envergadura",
+                        "nombre_largo": "Media wingspan",
```
```suggestion
                        "nombre_largo": "Half wingspan",

```

devarops commented on data/raw/datapackage.json:
```diff
@@ -205,9 +205,9 @@
+                        "name": "wingspan",
                         "long_name": "Wingspan",
-                        "nombre_largo": "Envergadura",
+                        "nombre_largo": "wingspan",
```
```suggestion
                        "nombre_largo": "Wingspan",

```

devarops commented on data/raw/datapackage.json:
```diff
@@ -157,9 +157,9 @@
+                        "name": "tarsus",
                         "long_name": "Tarsus",
-                        "nombre_largo": "Tarso",
+                        "nombre_largo": "tarsus",
```
```suggestion
                        "nombre_largo": "Tarsus",

```

devarops commented on data/raw/datapackage.json:
```diff
@@ -133,7 +133,7 @@
                     },
                     {
-                        "name": "altura_pico",
+                        "name": "beak_height",
```
```suggestion
                        "name": "bill_depth",

```

devarops commented on data/raw/datapackage.json:
```diff
@@ -97,7 +97,7 @@
                     },
                     {
-                        "name": "longitud_pico",
+                        "name": "beak_length",
```
```suggestion
                        "name": "bill_length",

```

devarops commented on data/raw/datapackage.json:
```diff
@@ -85,7 +85,7 @@
                     {
-                        "name": "longitud_craneo",
+                        "name": "skull_length",
                         "long_name": "Cranial length",
```
```suggestion
                        "long_name": "Head length",

```

devarops reviewed (changes requested):
Aquí te dejo unos comentarios

Commit:
⚗️ Modify function for different paths

Commit:
🚧 Add data set for tests

Se agregó un archivo de datos a la carpeta `tests/data/` donde se pusieron datos aleatorios de `data/raw/laysan_albatross_morphometry_guadalupe.csv`

devarops commented on data/raw/datapackage.json:
```diff
@@ -133,7 +133,7 @@
                     {
-                        "name": "altura_pico",
+                        "name": "beak_height",
                         "long_name": "Bill height",
```
```suggestion
                        "long_name": "Bill depth",

```

devarops commented on data/raw/datapackage.json:
```diff
@@ -85,7 +85,7 @@
                     },
                     {
-                        "name": "longitud_craneo",
+                        "name": "skull_length",
```
```suggestion
                        "name": "head_length",

```

devarops commented on data/raw/datapackage.json:
```diff
@@ -121,7 +121,7 @@
                     },
                     {
-                        "name": "ancho_craneo",
+                        "name": "skull_width",
```
```suggestion
                        "name": "head_width",

```

devarops commented on data/raw/datapackage.json:
```diff
@@ -121,7 +121,7 @@
                     {
-                        "name": "ancho_craneo",
+                        "name": "skull_width",
                         "long_name": "Craneal width",
```
```suggestion
                        "long_name": "Head width",

```

Commit:
⚗️Modify property names

Se cambió el orden de las propiedades en orden alfabetico en `/src/01_create_parameter_logistic_model_LAAL.R` al igual que se agregó una tolerancia de 1 en `/tests_01_create_parameter_logistic_model_LAAL.R`

Commit:
Merge remote-tracking branch 'origin/feature/variable_namechanges' into feature/variable_namechanges

devarops commented on tests/testthat/tests_01_create_parameter_logistic_model_LAAL.R:
```diff
@@ -39,10 +39,14 @@ test_that("El valor de prueba es correcto:", {
   expect_equal(obtained_json, json_correct_data)
+=======
+  expect_equal(json_data, json_correct_data, tolerance = 1)
+>>>>>>> Stashed changes
```
Aquí tienes conflictos de _merge_. Debes resolverlos.

devarops commented on src/01_create_parameter_logistic_model_LAAL.R:
```diff
@@ -4,13 +4,164 @@ library(tidyverse)
+<<<<<<< Updated upstream
 data_path <- "data/raw/"
 get_best_json_for_logistic_model(data_path, output_json_path)
+=======
```
Aquí tienes conflictos de _merge_. Debes resolverlos.

devarops commented on src/01_create_parameter_logistic_model_LAAL.R:
```diff
@@ -4,13 +4,164 @@ library(tidyverse)
+  threshold_error_table <- rbind(threshold_error_table, error_criteria)
+  setTxtProgressBar(progress_bar, i)
+}
+>>>>>>> Stashed changes
```
Aquí tienes conflictos de _merge_. Debes resolverlos.

devarops reviewed (changes requested):
¿Te están corriendo las pruebas con estos conflictos de _merge_?

Commit:
🔥 Remove stash code

Se eliminó codigo que interfería con el merge

Commit:
🔥 Remove stash code

Commit:
🎨 Improve format

Commit:
⚗️ Modify order in the test data

Se modificó el orden de los parámetros con sus respectivos valores en `tests/data/best_logistic_model_parameters_laag_ig_tests.json` y en `test/data/best_model_table_tests.csv`

Commit:
⚗️ Add a test function

Se agregó la función `get_best_json_for_logistic_model` para un mejor uso del *src*

Commit:
⚗️ Modify function paths

Commit:
🎨🐛 Improve format and debug

Commit:
🐛🚧 Add debug tests

Commit:
🎨 Improve format

Commit:
⚗️ Modify function

Se movio la funcion fuera del for numero 1

Commit:
🎨 Improve format

Commit:
⚗️ Modify function

Commit:
🔨🔥 Delete dependency

Se quitó la dependencia con el setup en `coverage`

Commit:
⚗️ Modify path order in tests

Commit:
🔨🔥 Delete dependency

Commit:
⚗️ Modify `fit_setpwise` function

Commit:
♻️ Refactor functions 

Refactorización de la función `get_best_json_for_logistic_model`, de las cuales se hicieron 3: `make_empty_dataframe`, `make_null_modeltable`, `rename_model_table`

Commit:
🎨 Improve format

Commit:
🧪 Add tests

Commit:
♻️ Refactor functions

Commit:
🧪 Add tests

Commit:
⚗️🎨 Modify function and improve format

Commit:
⚗️🧪 Modify test and add tests data

Commit:
♻️ Refactor functions

Commit:
⚗️🧪 Modify test

Commit:
⚗️ Modify function

Commit:
🧪 Add tests

Commit:
🎨 Improve format

Commit:
⚗️🧪 Modify test

Commit:
⚗️ Modify function

Commit:
🧪 Add tests

Commit:
⚗️🧪 Modify test

Commit:
⚗️🧪 Modify property name

*brim* to *wing*

Commit:
⚗️ Modify property name

Commit:
⚗️ Modify property name

mvillasante reviewed (commented):
LGTM

AlejandroPad commented:
🔀🐤 Cambia nombre de variables
- [x] Cambiamos a inglés los nombres de las propiedades de `tests/data/logistic_model_parameters_tests.json`. 
- [x] Detectamos que al cambiarlas, surge un error en las pruebas con los valores esperados (al intentar hacer lo análogo en `data/processed/logistic_model_parameters.json` al correr las pruebas se vuelven a cambiar automaticamente a su forma Spanglish). 
- [x] Probar con otro json file generado con los datos de prueba
## Hecho
- Extrajimos la función `get_best_json_for_logistic_model` 
- Factorizamos funciones pequeñas en `get_best_json_for_logistic_model`
## Pendientes
- [ ] Eliminar json de la receta de los tests en `Makefile`
- [ ] Extraer funciones de los `src`
